Gemfile

```diff
@@ -102,6 +102,7 @@ group :development, :test do
   gem 'factory_bot'
   gem 'factory_bot_rails'
   gem 'rails-controller-testing'
+  gem 'rails-flog', :require => 'flog'
```

Comment:
__[RuboCop]__ <sub>posted by Sider</sub>

Style/HashSyntax: Use the new Ruby 1.9 hash syntax.